llvm/include/llvm/CodeGen/SelectionDAG.h

```diff
@@ -229,6 +230,7 @@ class SelectionDAG {
   const TargetLibraryInfo *LibInfo = nullptr;
   const FunctionVarLocs *FnVarLocs = nullptr;
   MachineFunction *MF;
+  MachineFunctionAnalysisManager *MFAM = nullptr;
```

Comment:
`SDAGISelPass` is only used in `SIISelLowering.cpp`, I don't think we need this for now